admin-dev/themes/new-theme/js/pages/product/combination/bulk-form-handler.ts

```diff
@@ -45,19 +45,20 @@ export default class BulkFormHandler {
 
   private tabContainer!: HTMLDivElement;
 
+  private bulkChoicesSelector: BulkChoicesSelector;
+
   constructor(productId: number) {
     this.productId = productId;
     this.combinationsService = new CombinationsService();
     this.eventEmitter = window.prestashop.instance.eventEmitter;
-
     this.tabContainer = document.querySelector<HTMLDivElement>(CombinationMap.externalCombinationTab)!;
+    this.bulkChoicesSelector = new BulkChoicesSelector(this.tabContainer);
+
     this.init();
   }
 
   private init(): void {
-    this.listenCheckboxesChange();
-    this.eventEmitter.on(CombinationEvents.listRendered, () => this.toggleBulkActions());
-
+    //@todo; what does this mean? why there are 2 almost same looking selectors?
```

Comment:
ping @jolelievre can you remind what is this about?

Reply:
No idea 😅. Seems like a mistake that no one noticed Maybe because the PR was made by different people It seems like only one button is enough here If it can be simplified here it's probably better